microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,15 +131,98 @@ You can access Collections from an InPrivate window when using a signed-out prof
 <!-- ====================================================================== -->
 ## Compose
 
-In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.
+The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.
+
+
+<!-- ------------------------------ -->
+#### Use Compose to draft text within the Edge Sidebar
+
+In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections:
 
 ![Compose UI](./index-images/compose.png)
```

Comment:
not relevant to privacy
```suggestion
```

Reply:
removed colon, png line, & png file.  result:
```
In the Copilot pane, the Compose tool uses AI 
to write for you, starting from a short prompt 
and **Tone**, **Format**, and **Length** selections.

Text that's generated by Compose can be copied, 
or inserted directly into the webpage.
```